docs/layers/identity/centralized-terraform-access.mdx

```diff
@@ -60,8 +218,55 @@ Follow the Identity Providers documentation for adding a SAML login.
 
 With AWS SAML, we create a federated SAML login that connects to the "team" in the identity account, and then users can assume other roles from there. We use the [AWS Extend Switch Roles plugin](https://github.com/tilfinltd/aws-extend-switch-roles) that makes this much easier, but it's not as intuitive as Identity Center.
```

Comment:
We should explain somewhere what we mean by federated

> Federated login means that instead of managing separate credentials for each AWS account, users authenticate through a centralized identity provider (IdP). This allows them to access multiple AWS accounts or services using a single set of credentials, based on trust relationships established between the IdP and AWS.